packages/paste-website/src/components/tokens-list/TokensExample.tsx

```diff
@@ -97,10 +126,27 @@ export const RadiiBox: React.FC<RadiiBoxProps> = ({borderRadius}) => {
 };
 
 interface ShadowBoxProps extends BoxProps {
+  boxColor: string;
   shadow?: string;
 }
-export const ShadowBox: React.FC<ShadowBoxProps> = ({shadow}) => {
-  return <Box borderRadius="borderRadius20" padding="space60" css={{boxShadow: shadow}} />;
+export const ShadowBox: React.FC<ShadowBoxProps> = ({boxColor, shadow}) => {
+  let isInverse = false;
+  if (boxColor.includes('inverse') || boxColor.includes('light') || boxColor.includes('lighter')) {
+    isInverse = true;
+  }
+  return (
+    <Absolute
+      backgroundColor={isInverse ? 'colorBackgroundBodyInverse' : 'colorBackgroundBody'}
+      padding="space60"
+      preset="fill"
+      css={{
+        display: 'flex',
+        alignItems: 'center',
+      }}
+    >
+      <Box borderRadius="borderRadius20" padding="space60" width="100%" css={{boxShadow: shadow}} />
```

Comment:
@SiTaggart Out of scope for this PR, but should we be wary of the `css` prop here?